CommonMark.Portable45/CommonMark.Portable45.csproj

```diff
@@ -0,0 +1,175 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
yes, it should have been updated to VS2015 some time ago... 
And the project file structure is a bit of magic - but it is the best that can be done currently to enable this scenario without duplication of project structure.